### PR TITLE
Add role-admin as a label

### PR DESCRIPTION
**Pull request by recrwplay**
Adds a label to be used in Neo4j procedures documentation, to mark procedures with `@Admin`

### CODE REVIEW

**Comment by recrwplay** [comment]
> from what I can see this will use any text you want as the label, and this PR is just creating the label styling?

Correct. This just applies the text and background colour for any `label:role-admin[]` usage.

File: preview-src/docs-roles.adoc [hunk 1]
```diff
@@ -84,3 +84,22 @@ Weighted
...
+| Name |
+
+| <<procedure_db_checkpoint, `db.checkpoint()`>>
+| +++
+<span class="label label--enterprise-edition">Enterprise Edition</span>
++++
+
+| <<procedure_db_clearquerycaches, `db.clearQueryCaches()`>>
+| +++
+<span class="label label--role-admin">@Admin</span>
```
Comment (lidiazuin) [inline preview-src/docs-roles.adoc:102]: After discussion with @davidoliverSP2 , we decided "Admin only" would be a better title for the label.

```suggestion
<span class="label label--role-admin">@Admin</span>
```

**Review by davidoliverSP2 — APPROVED** [review]
thanks @lidiazuin & @recrwplay 

I think this works with Lidias amendment of the label text being `Admin only`

File: src/css/labels.css [hunk 2]
```diff
@@ -118,3 +118,8 @@
   background: var(--color-yellow-300);
   color: var(--color-black);
 }
+
+.label--role-admin {
```
Comment (recrwplay) [inline src/css/labels.css:122]: If we change this rule to match `.label--role-admin-only` you will get, by default, 'Admin Only' as the label text. That is what you want, bar the headline style caps.

**Comment by recrwplay** [comment]
Just to clarify, the text can be anything you like - the instance given as an example in this PR is just for illustration.

As it stands, you will need to use the label with `label:admin[Admin only]` to get the desired output.

If we were to change the css rule to match for `label:admin-only[]` you could have 'Admin Only' as the default text.

File: src/css/labels.css (hunk 2 above)
Comment (lidiazuin) [reply src/css/labels.css:122]: Got it! Thanks Neil. Please can you change this to `.label--role-admin-only` then? :)

**Review by lidiazuin — APPROVED** [review]

**Pull request merged**